index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
```

Comment:
It would be very useful to add some more metas that help with mobile-version scaling for example